broker/src/main/java/io/zeebe/broker/Broker.java

```diff
@@ -218,7 +219,6 @@ private StartProcess initStart() {
             return embeddedGatewayService;
           });
     }
-    startContext.addStep("cluster services", () -> atomix.start().join());
     startContext.addStep("topology manager", () -> topologyManagerStep(clusterCfg, localBroker));
```

Comment:
Could we please move it after the topology manger :)

Reply:
@Zelldon Done :)